pkg/config/cmd_config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+)
+
+type CmdContextParamCodegen struct {
+	Transform string
```

Comment:
this struct should have yaml tags

And this config should be with the codegen code, not in a seperate package.

Reply:
As Josh has updated code and moved types to package cmdutils, I removed CmdContextParamCodegen here and use the one in cmdutils, also added yaml tags.